### ISSUE TITLE
CT: cypress doesn't recognize "+" symbol in the path to the Angular component

### ISSUE THREAD

**Issue by bobige** [cypress-io/cypress#29493]
### Current behavior

Importing an angular component into a test file, when the component folder name starts with a "+" symbol, is not possible.

### Desired behavior

Cypress imports components that have the "+" symbol at the start of the containing folder name.

### Test code to reproduce

1. Create a new Angular 15 project.
2. Create a component in a folder that has the "+" sign in front of it
3. Create a basic test file for the component and just mount it.

### Cypress Version

13.9.0

### Node version

v16.18.1

### Operating System

Windows 10.0.19045 Build 19045

### Other

I am configuring Cypress to test Angular 15 components. We have a naming convention to put the "+" sign in front of the folder name, containing the component files. When I mount the component, cypress gives a "No tests found" error, which according to the documentation could be caused by a syntax error.

It is easily reproduceable. Create a new Angular 15 project and create a component in a folder that has the "+" sign in front of it. I also use Chrome 124:

![image](https://github.com/cypress-io/cypress/assets/81616281/9c309adc-e1ea-4f66-bae6-fb4ee1115455)

If I remove the "+" sign or even change the naming to have a different special character, such as "_", it works:

![image](https://github.com/cypress-io/cypress/assets/81616281/cc061c99-7649-4c47-916b-934f32d01055)

**Comment by H0onnn** [comment]
@jennifer-shehane Hello ! Can I solve this problem ?

**Comment by jennifer-shehane** [comment]
@H0onnn Yes, please follow our contributing guidelines and write a test for the behavior and we could review.

**Comment by H0onnn** [comment]
@jennifer-shehane Hello, is there a way to check if the test works properly with the modified code in my local environment?

**Comment by bobige** [comment]
@H0onnn @jennifer-shehane Hello, is there any progress on this issue?

**Comment by jennifer-shehane** [comment]
@bobige No, this is open to contributors if someone wants to make a PR though
